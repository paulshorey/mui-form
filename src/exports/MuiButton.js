/*eslint no-unused-vars: 2*/
import React from 'react';

// import { isEqualShallow, get_property } from 'functions';
import * as Styled from './MuiButtonStyled';

/*
	export: button
	* not used in MuiForm - only serves as export for Storybook
*/
export class MuiButton extends React.Component {
	render() {
		const { children, ...attributes } = this.props;
		return <Styled.Button {...attributes}>{children}</Styled.Button>;
	}
}

export default MuiButton;
