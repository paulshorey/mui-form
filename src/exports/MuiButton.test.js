import 'window/theme';

import React from 'react';
import renderer from 'react-test-renderer';

// shallow + redux
import { MuiButton } from './MuiButton';

it('renders correctly', () => {
	const tree = renderer
		.create(
				<MuiButton><b>Save Changes</b></MuiButton>
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
