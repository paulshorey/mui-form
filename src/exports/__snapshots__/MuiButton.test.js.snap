// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="sc-bdVaJa dnAqev"
>
  <b>
    Save Changes
  </b>
</button>
`;
